src/sagemaker_huggingface_inference_toolkit/handler_service.py

```diff
@@ -160,6 +160,21 @@ def postprocess(self, prediction, accept):
         """
         return decoder_encoder.encode(prediction, accept)
 
+    def transform_fn(self, model, input_data, content_type, accept):
+        # run pipeline
```

Comment:
could you add a proper `docstring` with args? We should also add an additional test for custom modules, which only implements `transform_fn` to confirm that it works properly